components/resource/stats.cpp

```diff
@@ -10,16 +14,67 @@
 #include <osgViewer/Viewer>
 #include <osgViewer/Renderer>
 
+#include <components/misc/stringops.hpp>
+
 namespace Resource
 {
 
+bool findFile(const boost::filesystem::path & dirPath, const std::string& fileName, boost::filesystem::path& pathFound)
+{
+    if (!boost::filesystem::exists(dirPath))
+        return false;
+
+    boost::filesystem::directory_iterator endIt;
+    for (boost::filesystem::directory_iterator it(dirPath); it != endIt; ++it)
+    {
+        const std::string lowerName = Misc::StringUtils::lowerCase(it->path().filename().string());
+
+        if (boost::filesystem::is_directory(*it))
+        {
+            if (findFile(*it, fileName, pathFound))
+                return true;
+        }
+        else if (lowerName == fileName)
+        {
+            pathFound = *it;
+            return true;
+        }
+    }
+    return false;
+}
+
+std::string getFont()
+{
+    std::string fontName = "arial.ttf";
+
+#if defined(_WIN32)
+    boost::filesystem::path fontPath = "C:/windows/fonts";
```

Comment:
May be use "%WINDOWS%/Fonts" (and code for expanding variables)? Users which installing system to non-standard locations won't be upset

Reply:
Maybe a good idea, but I do not use Windows and can not test if it will work or not.
Does boost::filesystem::path understand OS-specific environmental variables?

Reply:
Also I doubt that many people will complain - "C:/windows/fonts" is harcoded in OSG code and we used it for years.

Reply:
You can use SHGetFolderPathW with CSIDL_FONTS set as CSIDL for determining the path to user fonts folder on Windows, like windows-specific paths are [already](https://github.com/OpenMW/openmw/blob/master/components/files/windowspath.cpp) determined.

Reply:
I guess I'll leave it for Windows developers.